default

```diff
@@ -0,0 +1,22 @@
+server {
+	listen 80 default_server;
+	listen [::]:80 default_server;
+
+	root /var/www/html/wordpress;
+
+	index index.php index.html index.htm;
+
+	#server_name localhost;
+	server_name 192.168.50.2;
```

Comment:
what is the reason to use `server_name` ?

Reply:
`server_name` points to the `url` specified when setting up wordpress `wp core install --url="192.168.50.2"`. 

At first I didn't specify this property but routes and theme of wp didn't work.